tests/integration/test_charm.py

```diff
@@ -148,6 +237,7 @@ async def test_scale_up(ops_test: OpsTest):
     )
 
 
+@pytest.mark.usefixtures("only_without_juju_secrets")
```

Comment:
As indicated in the corresponding, specific commit: this is a temporary change until https://github.com/canonical/mongodb-k8s-operator/pull/176 is merged. I'll rebase on that, wanna have relevant pipelines until.